client/Makefile

```diff
@@ -6,3 +19,18 @@ develop: clean
 
 clean:
 	git clean -fdX
+
+# Targets for workstation
+install-workstation: clean uninstall
+	$(PREFIX) $(PIP) install --no-deps ./
+
+uninstall:
+	$(PREFIX) $(PIP) uninstall -y $(PACKAGE)
+
+develop-install: clean develop-uninstall
+	$(PREFIX) $(PIP) install --no-deps -e ./
```

Comment:
I think it is better to not use $(PREFIX) here, since setuptools+pip do not handle well editable installations under `/usr/local/lib...`